api/src/test/java/org/openmrs/DrugOrderTest.java

```diff
@@ -314,4 +314,95 @@ public void setAutoExpireDateBasedOnDuration_shouldNotCalculateIfAutoExpireDateA
 		
 		assertEquals(expectedAutoExpireDate, drugOrder.getAutoExpireDate());
 	}
+
+	/**
+	 * @verifies return true if the drugs match and drug non coded match
+	 * @see DrugOrder#hasSameOrderableAs(Order)
+	 */
+	@Test
+	public void hasSameOrderableAs_shouldReturnTrueIfTheConceptsMatchAndHaveSameDrugNonCoded() throws Exception {
```

Comment:
Actually this should be if concepts are both the DRUG OTHER  and and not just any concept